### PR TITLE
allow secrets to be set from env vars

### PR DESCRIPTION
**Pull request by krancour**
This is a partial solution to #42, mainly for the sake of discussion.

### CODE REVIEW

**Comment by krancour** [comment]
I'm moving this out of draft because I'm fairly convinced this moves the ball down court.

File: cmd/drake/secrets_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			input:          "${SUB}bar",
+			expectedResult: "foobar",
+		},
+		{
+			name: "multiple substitutions",
+			setup: func() {
+				os.Setenv("SUB1", "foo")
+				os.Setenv("SUB2", "bar")
+			},
+			input:          "${SUB1}${SUB2}",
```
Comment (carolynvs) [inline cmd/drake/secrets_test.go:41]: You may want a test case for repeated substitutions, e.g. "${SUB}${SUB}"

Comment (krancour) [reply cmd/drake/secrets_test.go:41]: Sure. I can add that.

**Review by carolynvs — APPROVED** [review]
LGTM

**Pull request merged**